h/static/scripts/tests/util/string-test.js

```diff
@@ -18,4 +18,56 @@ describe('util/string', function () {
       assert.equal(stringUtil.unhyphenate('-foo-bar-baz'), 'FooBarBaz');
     });
   });
+
+  describe('stringUtil helpers', function(){
+
+    it('removes hungarian marks', function(){
```

Comment:
Note to self: These were ported across from the tests in `unicode-test.coffee` in the client.